velox/serializers/tests/PrestoSerializerTest.cpp

```diff
@@ -1545,6 +1544,19 @@ TEST_F(PrestoSerializerTest, deserializeSingleColumn) {
     assertEqualVectors(vector, deserialized);
   };
 
+  auto testSerializeSingleColumnRoundTrip = [&](const VectorPtr& vector) {
+    std::ostringstream output;
+    serde_->serializeSingleColumn(vector, nullptr, pool_.get(), &output);
+    const auto serialized = output.str();
+
+    auto byteStream = toByteStream(serialized);
```

Comment:
Abstract this common code between the 2 lamdbas as a single function and re-use to indicate that the deserializeSingleColumn is with the same function.

Reply:
Thanks @aditi-pandit, updated accordingly. Could you please take another look?